fictadvisor-web/src/app/(main)/schedule/schedule-page/calendar-section/components/groups-dropdown/GroupsDropDown.tsx

```diff
@@ -15,23 +16,32 @@ export const GroupsDropDown: FC<DropDownSectionProps> = ({ groups }) => {
   }));
   const setGroupId = useSetGroupId();
 
+  const memoGroups = useMemo(
+    () =>
+      groups.length
+        ? groups?.map(group => ({
```

Comment:
so, the groups cannot be undefined or any falsy value, why do you check groups?.map